### DISCUSSION TITLE
Data from web API is not showing in UI

### DISCUSSION THREAD

**Discussion by Emianna** [unoplatform/uno#17937] [General]
I am consuming web api from Uno application and I am able to get the data and able to show the same in the UI in debug mode. But in release mode I am not able to see the data in UI. I can see the API response in network session under Dev tools. The issue is only for web application not for windows. Can you help me here? 
@jeromelaban

**Comment by jeromelaban** [comment]
Thanks for the question, this is generally cause by linker configuration, you can find more information here: https://platform.uno/docs/articles/common-issues-all-ides.html#runtime-error-no-parameterless-constructor-defined-for-xxxx

**Reply by tomchoondal** [reply]
@jeromelaban, we did not understand what the exact issue in linker configuration. Can you specify more on this.? We are hosted the API locally on IIS.

We hosted the wasm application in IIS. The debug build is fine and api data is coming. But when the release build is hosted in the IIS we cannot see the data. Even the app running from VS in release mode not returning any data. 

In your documentation https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html you mentioned like use debug build for IIS hosting. Is there a particular reason for that. Or can I know the IIS hosting is only supporting debug build?
